lib/buildFactory.js

```diff
@@ -93,14 +95,16 @@ class BuildFactory extends BaseFactory {
             status: 'QUEUED'
         }, config);
 
+        delete modelConfig.apiUri;
```

Comment:
Probably better to remove this line and add 
`hoek.applyToDefaults({status: 'QUEUED'}, { sha }) ` to line 103